src/components/_classes/component/Component.unit.js

```diff
@@ -62,7 +62,7 @@ describe('Component', () => {
       bad: {
         value: 'testte',
         field: 'firstName',
-        error: 'First Name must be shorter than 6 characters.'
+        error: 'First Name must contain 5 or less characters.'
```

Comment:
This sentence seems weird to me.  I would rather it say "First Name must be 5 characters or less"